src/main/java/org/example/AthletesList.java

```diff
@@ -0,0 +1,55 @@
+package org.example;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AthletesList {
+    private List<Athlete> li;
+
+    public AthletesList() {
+        this.li = new ArrayList<>();
+    }
+
+    public void addAthlete(Athlete a){
+        li.add(a);
+    }
+    public void listAthletes(){
+        for(Athlete a: li){
```

Comment:
please add more descriptive names for the variables
```suggestion
        for(Athlete athlete : li) {
```